package.json

```diff
@@ -6,9 +6,9 @@
   "browser": "lib/index.js",
   "module": "index.js",
   "scripts": {
-    "prepublish": "npm run transpile",
-    "transpile": "mkdir -p lib && babel -o ./lib/index.js ./index.js",
-    "test": "mocha --spec reporter --compilers js:babel-register ./test.js"
+    "prepublishOnly": "npm run build",
```

Comment:
I'd suggest:
```
"prepare": "npm run build",
"prepublishOnly": "npm run test",
```

See:
https://docs.npmjs.com/misc/scripts#prepublish-and-prepare
https://github.com/npm/npm/issues/10074

Reply:
Yeah I'm well aware of the details of those different commands, but I also disagree. There is no need to transpile files again on install if those transpiled files are already bundled. It just increased install time for no actual value. If you need to run it again when running `npm link` it's probably better to have a separate `build:watch` running that rebuilds on each change.

I've never been a fan of `prepublish` running on install in the first place. It think I've actually opened an issue about it way back when. But the npm team couldn't get rid of it for legacy reasons. 

As for tests running on `prepublishOnly`, in general it's probably better to have something like Travis in place to run tests on each PR/merge.